### ISSUE TITLE
Alert when a new issue missing a time box 

### ISSUE THREAD

**Issue by noamoss** [RoySegall/nuntius-bot#28]
Account managers are required to assure to include timebox in the [Xh] structure.
Each Account manager is working/responsible over more than one repo.
If a new issues was created in a repo, nutius-bot would send me (account manager) the following:

> "Hey, The issue [<issue title>](link) was created without a timebox!"

**Comment by RoySegall** [comment]
One #30 will be done we can work on this one.

**Comment by RoySegall** [comment]
a d 10x for the issue.

**Comment by RoySegall** [comment]
@noamoss I've added basic integration with github. After #40 will get in we can discus how we want to do that.
